core/src/main/java/com/graphhopper/routing/util/spatialrules/SpatialRuleLookupArray.java

```diff
@@ -105,10 +115,10 @@ public SpatialRule lookupRule(double lat, double lon) {
             }
         }
 
-        return EMPTY_RULE;
+        return EMPTY;
     }
 
-    protected int getRuleIndex(int xIndex, int yIndex) {
+    protected int getRuleContainerIndex(int xIndex, int yIndex) {
         if (xIndex < 0 || xIndex > lookupArray.length) {
```

Comment:
Yeah, let's just leave it as is. If we see any performance issues we can still tweak?